### PR TITLE
added logic to open 2049 for certain groups.

### PR DESCRIPTION
**Pull request by scb6**

### CODE REVIEW

File: roles/iptables/templates/etc/iptables [hunk 1]
```diff
@@ -29,6 +29,9 @@
 {% for ipaddr in groups.legacy_frontend -%}
 -A INPUT -p tcp -m tcp -m multiport --dports 8080,8888,8991 -m comment --comment "allowed TCP ports for Legacy" -j ACCEPT
 {% endfor -%}
+{% for ipaddr in [groups.legacy_frontend, groups.frontend, groups.archive, groups.pdf_gen] -%}
```
Comment (reedstrm) [inline roles/iptables/templates/etc/iptables]: Actually, I'm pretty sure all we need is `groups.nfs_connected` In fact, that is a super-set of the above groups, with in addition a couple I missed.

Comment (reedstrm) [reply roles/iptables/templates/etc/iptables]: Sorry I misled you initially :/

**Comment by reedstrm** [comment]
Ohh, and @DarkPrinceFrost  here's a handy way (using exactly this case as the example!) to have different stanzas for the different servers:
https://stackoverflow.com/questions/31347921/how-to-structure-a-template-file-which-might-be-different-for-groups-of-hosts-in

**Comment by dtwilliamson** [comment]
How is that different other than using `if` instead of `for`?

**Comment by reedstrm** [comment]
@dtwilliamson  - it'd be "if" one group then "for" over another: in this case, if nfs_server, loop over nfs_clients

**Pull request merged**